File: Calculadora Orientada a Objetos em Python/operacoes.py
# operacoes da calculadora em Python !

import soma
import subtracao
import divisao
import multiplicacao

class operacoes:
  def operar(Valor1, Valor2, entrada):
   if entrada == 1:
      final = resul = soma.soma.somar(Valor1, Valor2)

   elif entrada == 2:
      final = resul = subtracao.subtracao.subtrair(Valor1, Valor2)

   elif entrada == 3:
      final = resul = divisao.divisao.dividir(Valor1, Valor2)

   elif entrada == 4:
      final = resul = multiplicacao.multiplicacao.multiplicar(Valor1, Valor2)
   else:
       print("Escolha Invalida!!")

   return final

File: Calculadora Orientada a Objetos em PHP/Operacoes.php
<?php
#Inclui as Classes
include("Soma.php");
include("Subtracao.php");
include("Divisao.php");
include("Multiplicacao.php");

#Cria a classe operações
class Operacoes
{  
   private $resul;
   private $final;
    #Cria a função operar que recebe valor 1, valor2 e a escolha da operação.
    public function operar($Valor1, $Valor2, $entrada)
   {
     #Escolhe a operação a ser executada.
     switch ($entrada) {
      case '1':
          # code...
          $this->resul = new Soma;
          $this->final = $this->resul -> somar($Valor1,$Valor2);
          break;
      case '2':
          $this->resul = new subtracao;
          $this->final = $this->resul -> subtrair($Valor1,$Valor2);
          break;     
      case '3':
          # code...
          $this->resul = new divisao;
          $this->final = $this->resul -> dividir($Valor1,$Valor2);
          break;
      case '4':
          # code...
          $this->resul = new multiplicacao;
          $this->final = $this->resul -> multiplicar($Valor1,$Valor2);
          break;   
      default:
          # code...
          echo ("Escolha Invalida!!");
          break;
        }
        
        return $this->final;

    }
 
}